doc/doc-style-rules.md

```diff
@@ -0,0 +1,134 @@
+# Documentation style rules
+
+Thank you for helping us improve the documentation! Below we have described some rules that we
+advise you to follow when you expand the documentation with new articles or add new sections to it.
+
+Any comments should be written in the following format:
+
+```
+> **Note**
+> Text of
+> note
+```
+
+The same goes for Warning:
+
+```
+> **Warning**
+> Text of
+> your warning
+```
+
+Do not use `->`, `=>` instead of `–`:
+
+```
+`@FN` – name of the current V function
+```
+
+All filenames must be **bold**:
+
+```
+**file.v**
+```
+
+All symbols (modules, functions, structures, etc.) must be `monospaced`:
+
+```
+Module `http.net`
+```
+
+A single terminal command may be written inline, but must be `monospaced`:
+
+```
+Command `v install`
+```
+
+Several terminal commands must be wrapped in \`\`\`:
+
+````
+```
+cd project
+v install markdown
+```
+````
+
+The language name must be capitalized:
+
+```
+V
+```
+
+Every sentence must end with a punctuation mark:
+
+```
+Please see the Troubleshooting section on our wiki page.
+```
+
+Use — or – instead of - when grammar requires it:
```

Comment:
No. Always use `-` .

Reply:
The rest of the dashes can go die in typographical obscurity.

Reply:
The rules for them are hard to understand, vary between languages, and they are also hard to type on a regular keyboard. Showing them in terminals also often looks much uglier than just the minus `-` sign, because of the used fonts, that are often tweaked for readability in programming.